cli/src/cortx_prep.sh

```diff
@@ -70,11 +70,20 @@ gpgcheck=0
 name=Repository Base
 enabled=1
 EOF
+
+        touch /etc/pip.conf
+cat <<EOF >/etc/pip.conf
+[global]
+timeout: 60
+index-url: $CORTX_RELEASE_REPO/python_deps/
+trusted-host: $CORTX_RELEASE_REPO
```

Comment:
does that make sense for local python index? In our states [we consider](https://github.com/Seagate/cortx-prvsnr/blob/197c8e4df5196c5c6ea88c5b47cc8813dd55c397/api/python/provisioner/srv/salt/repos/pip_config.sls#L27-L29) that only for remote ones

Reply:
Yes @andkononykhin2, it's a new requirement that follows the cortx-preq rpm from RE.